src/modsim2/data/loader.py

```diff
@@ -365,29 +361,40 @@ def _split_indices(
             cifar=self.cifar,
         )
 
-    def compute_similarity(self, only_train: bool = False):
+    def compute_similarity(
+        self, only_train: bool = False, metric_seed: int = 42
+    ) -> Dict:
         """
         compute similarity between data of A and B
         only_train removes the validation data from this comparison
         """
-
         # coerce data into single tensor (not subset)
         # TODO issue-15 want to get data post-transform
```

Comment:
We missed this in past PRs but issue #15 has been resolved for a while so this comment should be removed